controller/articles.controller.js

```diff
@@ -11,12 +11,23 @@ const getArticleId = (req, res, next) => {
         if (article) {
             res.status(200).send(article);
         } else {
-            res.status(404).send({status: 404, msg: 'Article not found'})
+            res.status(404).send({status: 404, msg: 'Unable to find article'})
         }
     })
     .catch((error) => {
         next(error)
     });
 };
 
-module.exports = { getArticleId }
\ No newline at end of file
+const getArticles = (req, res, next) => {
+    const body = req.body
```

Comment:
Why would there be a req.body for a get request? This is only for post requests.